app/services/activity/projects_for_report_finder.rb

```diff
@@ -0,0 +1,18 @@
+class Activity
+  class ProjectsForReportFinder
+    attr_accessor :report, :scope
```

Comment:
Is there are reason for these to be i) writers and ii) public? 

Keeping them private and immutable would I think be best unless there's a good reason not to?

Reply:
Yep, fixed that 👍